private/buf/cmd/buf/command/breaking/breaking.go

```diff
@@ -219,6 +219,11 @@ func run(
 		if flags.ExcludeImports {
 			breakingOptions = append(breakingOptions, bufcheck.BreakingWithExcludeImports())
 		}
+		if len(imageWithConfig.PluginConfigs()) > 0 {
+			if err := bufcli.CheckBetaPluginEnabled(container); err != nil {
```

Comment:
There needs to be a `TODO` to remove this check here and in `lint`.

Reply:
I thought this is fine as there is a TODO at the definition for `CheckBetaPluginEnabled`. Once the function is deleted, we will be reminded to remove it because it won't build otherwise, but now I think it's easier to spot the TODO here than in `bufcli/env.go`, going to change.

Reply:
addressed in [ba3180b](https://github.com/bufbuild/buf/pull/3288/commits/ba3180b3c61d5e51d7f10d83e8721e736d187997)